litestar/_signature/models/base.py

```diff
@@ -17,6 +20,7 @@
 class ErrorMessage(TypedDict):
     key: str
     message: str
+    source: NotRequired[Literal["cookie", "body", "header", "query"]]
```

Comment:
Why not required?

Reply:
I opted to not include dependencies in the source handling since it's not part of the user request, but it probably makes sense to include that, too. Do you agree? If so, are there perhaps other scenarios I'm not considering that we'd want to include? It's probably better to be comprehensive.

Reply:
I think you were correct to not include dependencies. Clients shouldn't know, or care, where a 500 originates from within the application as they cannot do anything about it, and so it would just unnecessarily leak implementation detail.

Reply:
Fair point.